umm-spec-lib/src/cmr/umm_spec/xml_to_umm_mappings/iso19115_2/spatial.clj

```diff
@@ -4,7 +4,8 @@
             [cmr.umm-spec.iso19115-2-util :as iso]
             [cmr.common.xml.simple-xpath :refer [select text]]
             [cmr.common.xml.parse :refer :all]
-            [cmr.spatial.encoding.gmd :as gmd]))
+            [cmr.spatial.encoding.gmd :as gmd]
+            [cmr.umm-spec.xml-to-umm-mappings.iso-shared.distributions-related-url :as sdru]))
```

Comment:
Also the namespaces should go on a separate line from the require and be alphabetized. It's all standards from here: https://stuartsierra.com/2016/clojure-how-to-ns.html